lib/lager.ex

```diff
@@ -52,6 +52,14 @@ defmodule Lager do
   defp num_to_level(_), do: nil
 
   defp log(level, format, args, caller) do
+    log(should_log(level), level, format, args, caller)
+  end
+
+  defp log(false, _level, _format, _args, _caller) do
+    nil
```

Comment:
I guess we should put format and args statements here before nil so that we remove the warnings